projects/packages/search/src/dashboard/components/pages/sections/plan-usage-section.jsx

```diff
@@ -188,6 +201,14 @@ const AboutPlanLimits = () => {
 							className="support-link"
 						/>
 					),
+					jpExternalIcon: (
+						<ExternalLink
+							href="https://jetpack.com/support/search/"
+							rel="noopener noreferrer"
+							target="_blank"
+							className="support-link"
+						/>
+					),
```

Comment:
Could this be combined with `jpPlanLimits`? There's a bit of code duplication here.

Reply:
This was the only way I could figure to do it to get the link the way I wanted it. If I combine them, the link text would be either:

1. "record indexing and request limits.(icon)" or…
2. "record indexing and request limits(icon)" with the period after the icon.

Or we drop the period. Looking at the designs again, I'm seeing we do not specify a period there so I'll give that a try.